pyproject.toml

```diff
@@ -40,6 +40,12 @@ dependencies = [
   "pathvalidate",
 ]
 
+[tool.uv]
+dev-dependencies = [
+    "pytest>=7.0",
+    "pytest-asyncio>=0.23.0",
+]
+
```

Comment:
out of curiosity, why normal dependencies actually don't specify a version they should be higher/lower?

Reply:
I think you could but in this case we don't have strong reason to?